### PR TITLE
HTML auto direction for specific textual content

### PR DESCRIPTION
**Pull request by verbiricha**

### CODE REVIEW

File: src/Pages/ProfilePage.tsx [hunk 1]
```diff
@@ -150,10 +150,12 @@ export default function ProfilePage() {
 
   function bio() {
     return (
-      aboutText.length > 0 && (
-        <>
-          <div className="details">{about}</div>
-        </>
+      aboutText.length && (
+        <React.Fragment>
```
Comment (v0l) [inline src/Pages/ProfilePage.tsx:154]: Fragment not needed

**Comment by verbiricha** [comment]
updated the onboarding flow to use automatic lang direction and removed unneeded fragment.

**Comment by v0l** [comment]
Is this the correct way to do it? What about english should that not be ltr?

**Comment by v0l** [comment]
I mean it does seems to work: https://rtl.snort-social.pages.dev/e/note170sn3kcxxzk9unchk4zh7zatzxu7pjgkn59hyqpt4h0yg454m7vsdrh7q2

**Comment by verbiricha** [comment]
auto will show text ltr or rtl depending on the content characters so does the right thing.

**Pull request merged**